Update direct_path_load.mdx (#7102)
EnterpriseDB/docs: product_docs/docs/epas/18/database_administration/02_edb_loader/invoking_edb_loader/direct_path_load.mdx

    -   You can't use SQL expressions in the data field definitions of the `INTO TABLE` clause.
    -   The `FREEZE` option isn't supported for direct path loading.

!!! Warning

>     Currently, a direct path load toward TDE database shall not be performed, because it will cause permanent and unrecoverable damage to the relation files in data-block level. 

Reviewer: Here too, let's see if we can add the subject of this action: 

```suggestion
    Don't use a direct path to load data into the TDE database. This will cause permanent and unrecoverable damage to the relation files at the data-block level. 
```

Author: Hi,
Since I am not sure if I'm authorized to commit directly, let me leave a comment instead.
Yes, I agree to your change. Please go ahead to commit your change, as long as your team think it's ok.

Kind Regards,
Yuki Tei
